brew/vars.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

Reply:
I felt /bin/sh was better because it was more precise, faster to type and easier to read. see the advantages of /usr/bin/env sh. no strong feelings about it. I quickly surveyed the existing scripts and saw 3 flavors in use. the brew scripts at moment are simple and dont use exotic syntax so no worries

I'll change it